mesheryctl/internal/cli/root/app/onboard.go

```diff
@@ -45,6 +44,16 @@ mesheryctl app onboard -f ./application.yml -s "Kubernetes Manifest"
 * Usage of mesheryctl app onboard
 # ![app-onboard-usage](/assets/img/mesheryctl/app-onboard.png)
 	`,
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = `Usage: mesheryctl app onboard -f [filepath] -s [source type]
+Example: mesheryctl app onboard -f ./application.yml -s "Kubernetes Manifest"
+Description: Onboard application`
+
+		if file == "" && len(args) == 0 {
+			return fmt.Errorf("the path does not exist \n\n%v", errMsg)
```

Comment:
Let's improve the error message to be something along the lines of `file path or application name not provided`?

Reply:
Nice suggestion

Reply:
Thanks for the suggestions